server/lib/renderers/html.js

```diff
@@ -25,6 +25,12 @@ function renderHTML(data, template, options) {
 
   options = options || {};
 
+  // load local language for momentjs if possible
+  let languageDependencie = String().concat('moment/locale/', options.lang);
+  if (util.moduleExists(languageDependencie)) {
```

Comment:
Instead of `util.moduleExists()`, a more useful function would be `util.loadModuleIfExists()` that returns the module or `false`.

Still a pretty cool feature though.